package.json

```diff
@@ -2,27 +2,28 @@
   "name": "solid-client",
   "version": "0.22.4",
   "description": "Common library for writing Solid read-write-web applications",
-  "main": "./index.js",
+  "main": "./lib/index.js",
   "files": [
-    "index.js",
     "config.js",
     "lib",
-    "dist"
+    "dist",
+    "src"
   ],
   "scripts": {
+    "babel": "babel src -d lib",
     "build-full": "webpack --progress --config webpack.config.js --output-filename solid-client-full.js",
     "build-with-rdflib": "webpack --progress --colors --optimize-minimize --optimize-occurrence-order --optimize-dedupe --config webpack.config.js",
     "build-without-rdflib": "webpack --progress --colors --optimize-minimize --optimize-occurrence-order --optimize-dedupe --config webpack-no-rdflib.config.js",
     "build-qunit-resources": "npm run clean && mkdir -p dist/resources && npm run build-full && browserify -r ./test/resources/profile-minimal.js:test-minimal-profile -o dist/resources/test-minimal-profile.js && browserify -r ./test/resources/profile-private.js:test-minimal-prefs -o dist/resources/test-minimal-prefs.js",
-    "build": "npm run clean && mkdir dist && npm run build-with-rdflib && npm run build-without-rdflib",
+    "build": "npm run clean && mkdir dist && npm run babel && npm run build-with-rdflib && npm run build-without-rdflib",
     "clean": "rm -rf dist/",
-    "standard": "standard lib/*",
+    "standard": "standard src/*",
     "tape": "tape test/unit/*.js",
-    "test": "npm run standard && npm run tape",
+    "test": "npm run standard && npm run babel && npm run tape",
     "qunit": "npm run build-qunit-resources && open test/integration/index.html",
     "preversion": "npm test",
-    "postversion": "git push --follow-tags",
-    "prepublish": "npm run build"
+    "postpublish": "git push --follow-tags",
```

Comment:
My logic here was -- the tags only make sense in the context of publishing.
If the push is done post-publish, it means that if publish fails, a developer can roll back the versioning slightly easier (locally, without having to change history on the remote repo).